Assets/MixedRealityToolkit/Interfaces/InputSystem/Handlers/IMixedRealityGestureHandler.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// <summary>
     /// Interface to implement for generic gesture input.
     /// </summary>
-    public interface IMixedRealityGestureHandler : IEventSystemHandler
+    public interface IMixedRealityGestureHandler : IMixedRealityBaseInputHandler
```

Comment:
Why was this added?

Reply:
Have a look at the documentation of IMixedRealityBaseInputHandler and the implementation of OnGestureStartedWithAction() in MixedRealityInputSystem and let me know if that leaves you with any doubts.